### PR TITLE
Add machine pools link and type

### PR DESCRIPTION
**Pull request by nimrodshn**
cc: @igoihman @tzvatot @vkareh ptal

### CODE REVIEW

File: model/clusters_mgmt/v1/machine_pool_type.model [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+limitations under the License.
+*/
+
+// Representation of a machine pool in a cluster.
+class MachinePool {
+	// ID used to identify the cluster that this machinepool is attached to.
+	link Cluster Cluster
+
+    // The number of Machines (and Nodes) to create.
+	Replicas          Integer
```
Comment (igoihman) [inline model/clusters_mgmt/v1/machine_pool_type.model:23]: indent is off

Comment (nimrodshn) [reply model/clusters_mgmt/v1/machine_pool_type.model]: Thanks! done.

**Comment by nimrodshn** [comment]
@igoihman please re-TAL

**Pull request merged**